scrooge-sbt-plugin/src/main/scala/com/twitter/ScroogeSBT.scala

```diff
@@ -210,13 +221,16 @@ object ScroogeSBT extends Plugin {
   )
 
   val packageThrift = mappings in (Compile, packageBin) ++= {
-    (if (scroogePublishThrift.value) (scroogeThriftSources in Compile).value else Nil).map { file =>
+    (if ((scroogePublishThrift in Compile).value) (scroogeThriftSources in Compile).value else Nil).map { file =>
```

Comment:
The `.value` macro will be expanded into something involving `<<=` (or related operators—I'm not sure about the details). In any case looking up this value in a `mapping in …` task shouldn't make anything harder for SBT.